### PR TITLE
Feature/po i efficiency improvement

### PR DESCRIPTION
**Pull request by ipa-mig-mc**
The issue of "point of interest feature" that all the sensors were always being placed very close to the sensor's placement position, is fixed in this branch (for all three algorithms).

### CODE REVIEW

File: seneka_sensor_placement/common/include/seneka_sensor_placement/seneka_utilities.h [hunk 1]
```diff
@@ -84,7 +84,7 @@ namespace seneka_utilities
     // holds if the target lies within the forbidden area --
     bool forbidden;
     // holds the priority of the target
-    int priority;
+    mutable int priority;
```
Comment (mgruhler) [inline seneka_sensor_placement/common/include/seneka_sensor_placement/seneka_utilities.h:100]: why do you need mutable here? Doesn't that introduce problems?

**Comment by mgruhler** [comment]
Other than that seems fine to me. @ipa-frm what do you feel?

**Comment by ipa-mig-mc** [comment]
I too was thinking to ask you about that. I need it because, as you know, target_info_fix is a const and it has the "priority"/point of interest information.

Now, the poi_feature's efficiency issue (from my understanding) was caused because near the sensor origin, some how a lot of rays overlap and pass through the point of interest, hence such a sensor placement point gets evaluated as being a better one than those which are farther from the sensor origin. 

To avoid this, as soon as first ray hits the point of interest, i DELETE its priority. and then restore it at the end of updateTragetsInfoRaytracing function, i.e. after all rays have been checked.

For deleting the priority temporarily, I have used mutable.

Other ways I could think of avoiding this issue were:
-making a copy targets_info_fix and delete/restore priority info of the copy. But, this copying of a std::vector each time updateinforaytracing function is called, would drop performance quite a lot.
-another way would be to separate the "priority" info from targets_info_fix.

Let me know if you think mutable should really be avoided and some other solution should be used.

**Comment by mgruhler** [comment]
@ipa-mig-mc is this finished now?

**Comment by mgruhler** [comment]
Alright, looks good!

**Pull request merged**